workbench-core/accounts/src/services/costCenterService.test.ts

```diff
@@ -115,6 +118,99 @@ describe('CostCenterService', () => {
     });
   });
 
+  describe('list costCenters', () => {
+    const costCenterId = 'cc-someId';
```

Comment:
You'll want to be careful if you ever have async code in your setup; it leads to a potential race condition because of how jest initializes everything. [They recommend throwing any setup code in a `beforeEach()` or `beforeAll()` method to guarantee you don't have any race conditions.](https://jestjs.io/docs/setup-teardown#one-time-setup)

Reply:
updated